src/SimpleWallet/SimpleWallet.cpp

```diff
@@ -284,7 +291,8 @@ struct TransferCommand {
               logger(ERROR, BRIGHT_RED) << "payment ID has invalid format: \"" << value << "\", expected 64-character string";
               return false;
             }
-          } else if (arg == "-f") {
+          } 
+		  else if (arg == "-f") {
```

Comment:
What is the purpose of moving `else if (...` to a new line? Please, revert this change.